aexpect/remote.py

```diff
@@ -344,7 +344,8 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                  interface=None, identity_file=None,
                  status_test_command="echo $?", verbose=False, bind_ip=None,
                  preferred_authenticaton='password',
-                 user_known_hosts_file='/dev/null'):
+                 user_known_hosts_file='/dev/null',
+                 extra_cmdline=None):
```

Comment:
If I understand correctly, the `extra_cmdline` is now a string which means that you no longer need `None` (it was useful if we still keep this as a list). You could instead use the simpler string default `extra_cmdline=''`.

Reply:
Updated as suggested, thanks for heads up.